__tests__/api.test.js

```diff
@@ -77,6 +75,29 @@ describe('Articles', ()=>{
                 });
             });
     })
+    describe.only('GET /api/articles', ()=>{
+        test("returns an array of article objects with correct properties", ()=>{
+            return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then((response) => {
+                const articles = response.body.articles;
+                expect(articles.length).toBe(13);
+                expect(Array.isArray(articles)).toBe(true);
+                articles.forEach((article) => {
+                    expect(typeof article.author).toBe('string');
+                    expect(typeof article.title).toBe('string');
+                    expect(typeof article.article_id).toBe('number');
+                    expect(typeof article.topic).toBe('string');
+                    expect(typeof article.created_at).toBe('string');
+                    expect(typeof article.votes).toBe('number');
+                    expect(typeof article.article_img_url).toBe('string');
+                    expect(typeof article.comment_count).toBe('number');
+                    });
+            })
+        })
```

Comment:
We should check that it has been sorted, have a look at jest sorted for reference